megamek/src/megamek/client/Client.java

```diff
@@ -953,7 +1015,41 @@ protected void receiveEntities(Packet c) {
             cacheImgTag(e);
         }
     }
-
+    
+    /**
+     * Receives a force-related update containing affected forces and affected entities
+     */
+    @SuppressWarnings("unchecked")
+    protected void receiveForceUpdate(Packet c) {
+        Collection<Force> forces = (Collection<Force>) c.getObject(0);
+        Collection<Entity> entities = (Collection<Entity>) c.getObject(1);
+        for (Force force: forces) {
+            game.getForces().replace(force.getId(), force);
+        }
+        for (Entity entity: entities) {
```

Comment:
```suggestion
        for (Entity entity : entities) {
```

Reply:
Not my style.

Reply:
This isn't a style bit, but code accessibility. You separate out the ":" to ensure that the symbol isn't read as part of the word and then ignored, as it makes it into a separate word.